### PR TITLE
new function to delete several google cloud storage blobs at the same time

### PR DESCRIPTION
**Pull request by melvil-donnart**

### CODE REVIEW

File: lox_services/persistence/storage/storage.py [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
-from typing import Union
+from typing import Union, List
 
 from google.api_core.page_iterator import HTTPIterator
 from google.api_core import exceptions
 from google.cloud.storage import Client, Blob
 
 
 from lox_services.persistence.config import SERVICE_ACCOUNT_PATH
+from lox_services.persistence.storage import storage
```
Comment (natasazekic-loxsolution) [inline lox_services/persistence/storage/storage.py:13]: ?

File: lox_services/persistence/storage/storage.py [hunk 2]
```diff
@@ -205,3 +222,29 @@ def delete_file_from_url(url: str):
...
     delete_file_from_storage(bucket_name=bucket, blob_name=blob)
+
+
+def delete_multiple_files_from_storage(
+    bucket_name: str,
+    blob_names: List[str],
+):
+    """Removes existing files specified from the given bucket in Google Cloud Storage.
+    ## Arguments
+    - `bucket_name`: The name of the bucket where file is.
```
Comment (natasazekic-loxsolution) [inline lox_services/persistence/storage/storage.py:233]:
```suggestion
    - `bucket_name`: The bucket's name where the files are stored.
```

File: lox_services/persistence/storage/storage.py [hunk 3]
```diff
@@ -205,3 +222,29 @@ def delete_file_from_url(url: str):
...
+    - `source_file`: The path to the file to remove.
```
Comment (natasazekic-loxsolution) [inline lox_services/persistence/storage/storage.py:234]:
```suggestion
    - `blob_names`: List of files to be deleted.
```

File: lox_services/persistence/storage/storage.py [hunk 4]
```diff
@@ -205,3 +222,29 @@ def delete_file_from_url(url: str):
...
+
+    ## Example
+        >>> delete_file_from_storage("invoices_clients", "Helloprint/Invoices/UPS/1Z1234567890.pdf")
```
Comment (natasazekic-loxsolution) [inline lox_services/persistence/storage/storage.py:237]:
```suggestion
        >>> delete_multiple_files_from_storage("invoices_clients", ["Helloprint/Invoices/UPS/1Z1234567890.pdf"])
```

File: lox_services/persistence/storage/storage.py [hunk 5]
```diff
@@ -205,3 +222,29 @@ def delete_file_from_url(url: str):
...
+
+    ## Returns
+    Nothing
```
Comment (natasazekic-loxsolution) [inline lox_services/persistence/storage/storage.py:240]:
```suggestion
```

File: lox_services/persistence/storage/storage.py [hunk 6]
```diff
@@ -205,3 +222,29 @@ def delete_file_from_url(url: str):
...
+):
+    """Removes existing files specified from the given bucket in Google Cloud Storage.
+    ## Arguments
+    - `bucket_name`: The name of the bucket where file is.
+    - `source_file`: The path to the file to remove.
+
+    ## Example
+        >>> delete_file_from_storage("invoices_clients", "Helloprint/Invoices/UPS/1Z1234567890.pdf")
+
+    ## Returns
```
Comment (natasazekic-loxsolution) [inline lox_services/persistence/storage/storage.py:239]:
```suggestion
```

**Review by natasazekic-loxsolution — APPROVED** [review]
Waiting for you to add the function from connecting Main PR, otherwise good.

File: lox_services/persistence/storage/storage.py (hunk 1 above)
Comment (melvil-donnart) [reply lox_services/persistence/storage/storage.py:13]: Oops, imported to test on a separate file
Removed it ✅

**Comment by melvil-donnart** [comment]
@natasazekic-loxsolution Good to merge ✅

**Pull request merged**